### PR TITLE
Update locust loadtest

### PR DESCRIPTION
**Pull request by dzaslavskiy**
configure locust to listen on all interfaces to enble using envoy proxy
update test flow url
add route
update python runtime, black

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 21.11b1 # Update with 'pre-commit autoupdate'
+    rev: 22.1.0 # Update with 'pre-commit autoupdate'
```
Comment (folksgl) [inline .pre-commit-config.yaml:3]: If we're going to update, why not go to the latest `22.3.0`?

File: manifest.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ---
 applications:
   - name: locust
-    no-route: true
+    routes:
+    - route: identity-idva-monitoring-locust-((ENVIRONMENT)).apps.internal
```
Comment (folksgl) [inline manifest.yaml:5]: Confused by this. Why does our load testing app need a route assigned to it? No other apps should ever be trying to send traffic here.

**Review by folksgl — CHANGES_REQUESTED** [review]

File: manifest.yaml (hunk 2 above)
Comment (dzaslavskiy) [reply manifest.yaml:5]: in order to be able to access the ui over https

Comment (folksgl) [reply manifest.yaml:5]: Where is the access to this application coming from? Is this somewhere that we're setting up an SSH tunnel for and hitting locally? Or is another app going to be hitting `https://identity-idva-monitoring-localhost-prod.apps.internal:61443` from somewhere? I don't see the value in adding a route if we're using SSH to hit the app, but if it's another cf-app then I don't think I understand how we're utilizing this app.

Comment (dzaslavskiy) [reply manifest.yaml:5]: This route is only to allow hitting the app over https after opening an ssh tunnel.

**Review by folksgl — APPROVED** [review]

**Pull request merged**